src/styles/header.scss

```diff
@@ -6,7 +6,7 @@ $navbar-item-hover-background-color: transparent;
 
 $navbar-dropdown-border-top: none;
 $navbar-dropdown-radius: 0;
-$navbar-breakpoint: $tablet;
+$navbar-breakpoint: $desktop;
```

Comment:
I believe we can just remove this line altogether since the Bulma default breakpoint is already `$desktop`. Does that work?